modules/k8s-nmstate-removing-interfaces.adoc

```diff
@@ -0,0 +1,61 @@
+// This is included in the following assemblies:
+//
+// k8s-nmstate-configuring.adoc
+
+[id="removing-interfaces_{context}"]
+
+= Removing interfaces
+
+To remove a configured interface from the cluster's nodes, you must explicitly change the policy from `state: up` to `state: absent` and apply the policy.
+
+[IMPORTANT]
+.Removing a policy doesn't revert the state
```

Comment:
s/doesn't/does not
We use single asterisks to embolden items, not double asterisks per https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

I suggest putting "Removing a policy does not revert the state." inside the text box, not in that heading format